handlers/me.go

```diff
@@ -0,0 +1,42 @@
+package handlers
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	database "speedrun.website/db"
+	"speedrun.website/graph/model"
+	"speedrun.website/middleware"
+)
+
+func MeHandler(c *gin.Context) {
+	user, _ := c.Get(middleware.JwtConfig.IdentityKey)
+	db, err := database.GetDatabase()
+
+	// todo error handler or middleware?
+	if err != nil {
+		log.Println("Unable to connect to database", err)
+		c.Error(err)
+		c.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{
+			"message": err.Error(),
+		})
+		return
```

Comment:
I generally would prefer to always bubble errors up, so returning a non-nil error here, and settingg a return type of error (so we can do an `if err != nil` check above), but it's not necessary and this may be where the error handling should stop.

Reply:
We already are at the top lever in this function. The router needs `gin.HandlerFunc` types and those don't return anything so I'm not sure I understand what I want here. But I am open for better error handling indeed, maybe a middleware?

Reply:
Oh, I wasn't sure if they would bubble up higher or not, since I'm not familiar with Gin. Basically if we wanted to bubble them up to our main.go, or if that would happen by erroring here. This is a good place anyways. The comment came more from go methodology being:

Only error/panic (hard stop) on truly unexpected things, and otherwise inform and let the layer above handle what it wants to do.